petclinic-frontend/src/features/inventories/InventoriesListTable.tsx

```diff
@@ -395,12 +397,22 @@ export default function InventoriesListTable(): JSX.Element {
       >
         Delete All Inventories
       </button>
+
       <button
         className="add-inventory-button btn btn-success"
-        onClick={() => {}}
+        onClick={() => setShowAddInventoryForm(true)}
```

Comment:
I would suggest adding a refresh for the inventories, just like you did for inventory types. This way you don't have to manually refresh the page to see the new changes.

Reply:
I agree, I will look into it. Thank you.